app-config.json

```diff
@@ -1150,6 +1150,1735 @@
+                        "id": "https://abr.business.gov.au/ABN/View?abn=90664869327",
+                        "name": "Sample Company Pty Ltd.",
+                        "registeredId": "90664869327",
+                        "idScheme": {
+                          "type": ["IdentifierScheme"],
+                          "id": "https://id.gs1.org/01/",
+                          "name": "Global Trade Identification Number (GTIN)"
+                        },
+                        "description": "A leading electric energy storage company"
+                      },
+                      "auditor": {
+                        "type": ["Party"],
+                        "id": "https://abr.business.gov.au/ABN/View?abn=90664869327",
+                        "name": "Sample Company Pty Ltd.",
+                        "registeredId": "90664869327",
+                        "idScheme": {
+                          "type": ["IdentifierScheme"],
+                          "id": "https://id.gs1.org/01/",
+                          "name": "Global Trade Identification Number (GTIN)"
+                        },
+                        "description": "A leading electric energy storage company"
+                      }
+                    }
+                  ]
+                },
+                "constructData": {
+                  "mappingFields": [],
+                  "dummyFields": [],
+                  "generationFields": [
+                    {
+                      "path": "/eventID",
+                      "handler": "generateIdWithSerialNumber"
+                    }
+                  ]
+                },
+                "className": "json-form",
+                "style": {
+                  "margin": "40px auto",
+                  "paddingTop": "40px",
+                  "width": "80%"
+                }
+              }
+            },
+            {
+              "name": "CustomButton",
+              "type": "Submit",
+              "props": {}
+            }
+          ],
+          "services": [
+            {
+              "name": "processDigitalConformityCredential",
+              "parameters": [
+                {
+                  "vckit": {
+                    "vckitAPIUrl": "http://localhost:3332/v2",
+                    "issuer": "did:web:60c5-115-79-212-164.ngrok-free.app"
+                  },
+                  "digitalConformityCredential": {
+                    "context": [
+                      "https://jargon.sh/user/unece/ConformityCredential/v/0.5.0/artefacts/jsonldContexts/ConformityCredential.jsonld?class=ConformityCredential"
+                    ],
+                    "renderTemplate": [
+                      {
+                        "template": "<div>Digital Conformity Credential</div>",
+                        "@type": "WebRenderingTemplate2022"
+                      }
+                    ],
+                    "type": ["VerifiableCredential", "DigitalConformityCredential"],
```

Comment:
Hi, maybe we do not need `VerifiableCredential` in here, because we set it as default when issuing a credential
<img width="457" alt="image" src="https://github.com/user-attachments/assets/c07387bb-620b-4b08-b886-f2f1632b5ddd">
<img width="849" alt="image" src="https://github.com/user-attachments/assets/43be8d4c-c0fd-45cf-ad2e-88b827cc13ac">

Reply:
Updated, thank you